src/components/EmailInputComponent.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <label>Почта</label>
```

Comment:
It would be a good practice to encapsulate all language values in separate interchangeable modules to make it easy to implement localization in the future development of the project. However, if it makes the development too complicated right now, we may forget about that for the time being until MVP is finished.